inc/CrvHitInfoReco.hh

```diff
@@ -21,12 +21,13 @@ namespace mu2e
     float             time = -1; // average hit time
     float             PEs = -1;   //total number of PEs for this cluster
     std::array<float, CRVId::nLayers> PEsPerLayer = {-1};  // PEs per layer for this cluster
+    std::array<float, CRVId::nLayers * CRVId::nSidesPerBar> sidePEsPerLayer = {-1};// PEs per layer per side for this cluster
```

Comment:
Hi Andy. 

1. Could you point me to an example of what you mean? Do you mean insert a `NaN` element in the array instead of a zero?
2. I'll make a GitHub for them. 
3. II'll add a comment.

Reply:
Thanks, Sam. For (1), you can keep a zero but just add a flag like this to the struct: https://github.com/Mu2e/EventNtuple/blob/23a12de3750b4ba3b8f3b911ac7348c3ddceab41/inc/SimInfo.hh#L15 just to differentiate zero meaning "we measured 0 PEs" vs "we don't have a readout on this side so setting to 0".

And then for (2), it's fine to add it to this repo in either ```example-analysis-scripts/``` or if it's using pyutil put it in there

Reply:
Hi Andy. OK, so we set it to false in cases where there is no coincidence in cluster. Alternatively, we could add an int called `nCoincs`, which would tell you the number of coincidences in the cluster? That might give you more information and may be less ambiguous. Either way is fine with me.

Reply:
Sorry, I was imagining having a global bool for every coincidence. What you're proposing is to have another array with the same structure as `sidePEsPerLayer` which is filled with bools, is that correct? 

Also, I've included some plotting scripts. The Python one uses a` utils.py` file, which might be better off being included in `pyutils`.  Let me know what you think.

Reply:
Oh, I see where my misunderstanding is then. I thought one of these structs was being filled per end but obviously that's not right... I'm not sure an array of bools is the way to go either. Is there another way to identify single-ended modules? e.g. are they in specific sectors so that knowing the ```sectorType``` is enough? We might just need to leave this as-is for now then but we should keep this detail in mind in case analyzers get confused

Thanks for adding the plotting scripts. Work with Sophie to see if anything in your ```utils.py``` should be included in ```pyutils```

Reply:
I'll coordinate with Sophie. Would it be easier for now to make `PEs.py` a standalone script?

Reply:
I don't think it matters too much. I'm considering everything going into ```example-analysis-scripts/``` needs properly organizing at some point so it's up to you

Reply:
OK, can we leave `PEs.py` as it is for now? I'll talk to Sophie and come back to it once I know which pieces should be moved into `pyutils`. 

One other problem with a `singleEnded` flag based on `sectorType` is that I think that certain modules within the same sector are single ended. That said, I do think it would be nice to have. I'll talk to Ralf and see if I can work it out how to do it, although it might have to go in a separate PR.

Reply:
Thanks, Sam. I'll approve and merge this now